src/pages/home/report/ReportActionsList.tsx

```diff
@@ -426,13 +427,32 @@ function ReportActionsList({
                     });
                     return;
                 }
+                const index = sortedVisibleReportActions.findIndex((item) => keyExtractor(item) === action?.reportActionID);
+                if (action?.actionName === CONST.REPORT.ACTIONS.TYPE.REPORT_PREVIEW && index > 0) {
+                    reportScrollManager.scrollToIndex(index);
+                    setActionIdHighlightedTemporary(action?.reportActionID);
+                } else {
+                    reportScrollManager.scrollToBottom();
+                }
 
-                reportScrollManager.scrollToBottom();
                 setIsScrollToBottomEnabled(true);
             });
         },
-        [report.reportID, reportScrollManager, setIsFloatingMessageCounterVisible],
+        [report.reportID, reportScrollManager, setIsFloatingMessageCounterVisible, sortedVisibleReportActions, reportScrollManager],
     );
+
+    // Clear highlight report action after scroll and highlight
+    useEffect(() => {
+        if (actionIdHighlightedTemporary === '') {
+            return;
+        }
+
+        const timer = setTimeout(() => {
+            setActionIdHighlightedTemporary('');
+        }, 3000);
```

Comment:
use `const` here.

highlight it the same time as when we get back to report from a thread (to parent thread)